### PR TITLE
test: filterActionBlocks test

### PR DESCRIPTION
**Pull request by jianwei1**
# Description

### Issue

Test for filterActionBlocks.ts

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37071143/todos/7356069258)

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  parentOrder: 0,
+  locked: false,
+  nextBlockId: null,
+  children: [card]
+}
+
+describe('filterActionBlocks', () => {
+  it('should filter action blocks', () => {
+    const filteredActionBlocks = filterActionBlocks(step)
+    expect(filteredActionBlocks).toHaveLength(3)
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts:94]: We can remove

File: apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    expect(filteredActionBlocks).toContain(button)
+    expect(filteredActionBlocks).toContain(RadioOption1)
+    expect(filteredActionBlocks).toContain(RadioOption2)
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts:97]: can be added into one check, e.g. `.toEqual(values inside)`

File: apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    // should not contain RadioQuestionBlock or image
+    expect(filteredActionBlocks).not.toContain(radioQuestionBlock)
+    expect(filteredActionBlocks).not.toContain(image)
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts:101]: separate to their own test

File: apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts (hunk 1 above)
Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts:94]: .toHaveLength(3) <- can remove this check

File: apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  it('should filter action blocks', () => {
+    const filteredActionBlocks = filterActionBlocks(step)
+    expect(filteredActionBlocks).toEqual([button, RadioOption1, RadioOption2])
+  })
+
+  it('should filter out radioQuestionBlock and image block', () => {
+    const filteredActionBlocks = filterActionBlocks(step)
+    expect(filteredActionBlocks).not.toContain(radioQuestionBlock)
+    expect(filteredActionBlocks).not.toContain(image)
+  })
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts:101]: I reckon we change the description into something more general, this focuses on RadioQuestion and image specifically. can update the expect to just in one check too

Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/filterActionBlocks/filterActionBlocks.spec.ts:101]: we should probably add a test suite as well for blocks that do have actions but value is null

**Review by GeronimoJohn — APPROVED** [review]

**Review by GeronimoJohn — APPROVED** [review]
